tasks/connectors/github_code_scanning/github_code_scanning_task.rb

```diff
@@ -82,9 +87,18 @@ def run(opts)
           initialize_options
           initialize_client
 
-          @repositories.each do |repo|
-            endpoint = "/repos/#{repo}/code-scanning/alerts"
-            import_alerts(repo, endpoint)
+          if !@repositories.empty? && @organizations.empty?
```

Comment:
I don't know why we don't just scan both if either are present, but this should work I suppose.